Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -92,7 +92,7 @@ class VIEWER_EXPORT Viewer : public CGAL::Three::Viewer_interface {
   //!Set total number of depth peeling passes.
    void setTotalPass(int);
    void resetFov();
-   const QVector3D& scaler() const;
+   const QVector3D& scaler() const override;
```

Comment:
Q_DECL_OVERRIDE for consistency with all other functions

Reply:
> Q_DECL_OVERRIDE for consistency with all other functions

Well, no. We should move away from `Q_DECL_OVERRIDE` now that we require C++14, and use `override` everywhere. In Qt6 they no longer use `Q_DECL_OVERRIDE` either, even if the macro is still defined.

Reply:
I can change all `Q_DECL_OVERRIDE` to the `override` to be consistent and respect the standard.

Reply:
Let's do that in another pull-request. This one is fine.